### ISSUE TITLE
[FEAT]  Enhance accessibility

### ISSUE THREAD

**Issue by D4vidDf** [D4vidDf/HyperBridge#17]
When screen readers are enabled by the user, the app doesn't have different id and content description for the toggle options on the list of the app.

Add new strings to specify which app will be enabled or disabled.

Improve color scheme inside health check screen.
